hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,219 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: hypershift-dvo-selectorsyncset-template
+parameters:
+  - name: REGISTRY_IMG
+    required: true
+  - name: IMAGE_TAG
+    required: true
+  - name: IMAGE_DIGEST
+    required: true
+  - name: REPO_NAME
+    value: deployment-validation-operator
+    required: true
+  - name: DISPLAY_NAME
+    value: Deployment Validation Operator
+    required: true
+  - name: CHANNEL
+    value: "alpha"
+    displayName: OLM subscription channel
+    description: OLM subscription channel
+    required: true
+  - name: TARGET_ORG
+    displayName: Target organization
+    description: Target api.openshift.com organization to deploy to
+    required: true
+objects:
+  - apiVersion: hive.openshift.io/v1
+    kind: SelectorSyncSet
+    metadata:
+      annotations:
+        component-display-name: ${DISPLAY_NAME}
+        component-name: ${REPO_NAME}
+      labels:
+        managed.openshift.io/gitHash: ${IMAGE_TAG}
+        managed.openshift.io/gitRepoName: ${REPO_NAME}
+        managed.openshift.io/osd: "true"
+      name: deployment-validation-operator-hypershift
+    spec:
+      clusterDeploymentSelector:
+        matchExpressions:
+          - key: ext-hypershift.openshift.io/cluster-type
+            operator: In
+            values:
+              - management-cluster
+          - key: api.openshift.com/fedramp
+            operator: NotIn
+            values:
+              - "true"
+      resourceApplyMode: Sync
+      resources:
+        - apiVersion: policy.open-cluster-management.io/v1
+          kind: Policy
+          metadata:
+            annotations:
+              policy.open-cluster-management.io/categories: CM Configuration Management
+              policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+              policy.open-cluster-management.io/standards: NIST SP 800-53
+            name: deployment-validation-operator
+            namespace: openshift-acm-policies
+          spec:
+            disabled: false
+            remediationAction: enforce
+            policy-templates:
+              - objectDefinition:
+                  apiVersion: policy.open-cluster-management.io/v1
+                  kind: ConfigurationPolicy
+                  metadata:
+                    name: deployment-validation-operator
+                  spec:
+                    evaluationInterval:
+                      compliant: 2h
+                      noncompliant: 45s
+                    object-templates:
+                      - complianceType: MustHave
+                        objectDefinition:
```

Comment:
We need to repeat this for every resource below, because this `objectDefinition` doesn't have array-like value.